src/main/java/com/example/demo/movies/MovieController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.movies;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping (path = "api/v1/movies")
+@RequiredArgsConstructor
+public class MovieController {
+
+    private final MovieService movieService;
+
+    @GetMapping
+    public List<Movie> getMovies() {
+        return movieService.getMovies();
+    }
+
+    @PostMapping
+    public void addNewMovie(@RequestBody Movie movie) {
+        movieService.addNewMovie(movie);
+    }
+
+    @DeleteMapping (path = "{movieId}")
+    public void deleteMovie(@PathVariable("movieId") Integer movieId) {
+        movieService.deleteMovie(movieId);
```

Comment:
Check if the movie does not exist in the database and return a suggestive message